faqs/galaxy/datatype_tabular.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Indentifying and Formatting Tabular Dataset 
+area: Datatypes
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+---
+ 
+
+- A Tabular datatype is human readable and has tabs separating data columns.
+  - Note: tabular data is different from comma seperated data (.csv)
+1. Common tabular datatypes are `.bed`, `.gtf`, `.interval`, or `.txt`.
```

Comment:
I think that this is a clarification, but not a proper step, same with step 2

Reply:
Noted.